bindings/c/CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# Contributing
+1. [Contributing](#contributing)
```

Comment:
Please following the same style like other docs, use `- xxx` instead of hand writing `1. xxx`